# PROJECT_NAME

## Summary

## Methods

Standard methods are explained inside the report at the end.

## Publication policy

"The authors would like to thank Lorena Pantano of the PILM Bioinformatics Core, Picower Institute of Learning and Memory, Cambridge, MA for assistance with RNAseq analysis."

https://github.mit.edu/pages/PILM-bioinformatics/knowledgebase/admin/projects/

## Data

Temporarly in OpenMind `/om/user/lpantano/projects/PROJECT_NAME`


## Results

Shared dropbox: https://www.dropbox.com/home/projects/PI/PROJECT_NAME

Inside `reports` folders:

* MultiQC: Qc metrics for the analysis and all samples.
* YY: differential expression (DE) analysis for YY genotype
 * de.html: DE report
 * de/results/counts: expression at different normalization levels
 * de/results/differential_expression/comparison: DE results
* ZZ: differential expression analysis for mut genotype (same folder structure)
* `meta_de_results.csv.gz`: all genes results from DE analysis

# Resources

Functional analysis: Look at the tip of the week in this newsletter to know the options: https://github.mit.edu/pages/PILM-bioinformatics/secure-newsletters/2019/05/pilm-bioinformatics-newsletter-week-20-functional-analysis-do-it-right/
